tests/notebook_ui_tests/notebook.mitosheet.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { IJupyterLabPageFixture, expect, test } from '@jupyterlab/galata';
+import { awaitResponse } from '../streamlit_ui_tests/utils';
+import path from 'path';
+
+test.describe.configure({ mode: 'parallel' });
+test.use({ appPath: '', autoGoto: false })
```

Comment:
What are we actually using Galata for in this file? It kinda feels like we could just eliminate it and pretty much nothing would change, no?

Reply:
Actually we're using it for `page.contents.uploadFile` in the `beforeEach`.